README.md

```diff
@@ -1,4 +1,4 @@
-![Goldstar icon](https://i.gse.io/frontend/company/image_assets/goldstar-logo-blue-on-white-small.png)
+![Goldstar icon](https://res.cloudinary.com/goldstar/image/upload/e_colorize,co_rgb:fec200,w_300/v1542129455/Goldstar_HorizontalLockup_Black.png)
```

Comment:
Are you sure we want to serve this via cloudinary?  We'll get charged for each transformation. Should just be an image on i.gse.io IMO